migrate.md

```diff
@@ -0,0 +1,111 @@
+
+    There are some additional steps that the IBM support team needs to take on your behalf. Add the following information to the case:
+
+    - Region in which the service instance is hosted, such as Dallas or Frankfurt
+    - Resource group name
+    - Resource group ID
+
+      If you don't know the ID, open the {{site.data.keyword.Bluemix_notm}} command line interface (CLI) tool and enter the following command:
+
+      ```bash
+      ibmcloud resource groups
+      ```
+      {: codeblock}
+
+      The response shows the ID of the resource group. For more information about the CLI command, see [Working with resources and resource groups ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://cloud.ibm.com/docs/cli?topic=cloud-cli-ibmcloud_commands_resource#ibmcloud_commands_resource).
+
+      Give the support team up to 24 hours from the time you submit the case to complete the work they need to perform to support the Premium plan instance migration. There is no disruption of service during the migration process.
+
+      You only need to perform this step the first time you migrate a service instance for the Premium plan. When you click *Migrate* for other service instances that belong to the same plan, they will be migrated to the same resource group without requiring any involvement from the IBM support team.
+      {: important}
+
+The old (Cloud Foundry org-based) service instance that you migrated continues to be listed in the *Cloud Foundry Services* section of the {{site.data.keyword.Bluemix_notm}} Resource list, and is now shown as an *alias* of the new (resource group-based) version of the instance.
+
+The following image shows how a {{site.data.keyword.conversationshort}} service instance is listed after it is migrated.
+
+![Shows current service instance is now an alias of a resource-based instance](images/alias.png)
+
+For more information about aliases, see the [IBM Cloud Connections documentation ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://cloud.ibm.com/docs/resources/connecting_apps#what_is_alias).
+
+You must open the new, resource group-based version of the service instance to be able to access any tools that are associated with it. The new instance is listed in the *Services* section of the {{site.data.keyword.Bluemix_notm}} Resource list.
+
+If your service was syndicated originally, after the migration, it might be hosted in a different location than you expect. The URL of the migrated service indicates its location.
+{: note}
+
+## Authentication
+{: #migrate-auth-support}
+
+If you have existing applications that use basic authentication to access the service, then they can continue to pass a username and password to authenticate with the service instance after it is migrated. You can see the credential information from the **Service credentials** page of the alias.
```

Comment:
You can probably change `then they can` to just `they can`.  Also, should `credential information` be `credentials information`?